scripts/index.js

```diff
@@ -14,8 +14,8 @@ const { getReactScriptsVersion, isEjected } = require('../utils');
 const paths = isEjected ? importCwd('./config/paths') : importCwd('react-scripts/config/paths');
 const webpack = importCwd('webpack');
 const config = isEjected
-  ? importCwd('./config/webpack.config.dev')
-  : importCwd('react-scripts/config/webpack.config.dev');
+  ? importCwd('./config/webpack.config')('development')
```

Comment:
I'm afraid it will be a bit more complicated than that. If we do that, we break the backward compatibility with `react-scripts` previous version. You need to use the `react-scripts` version that are available to you higher up: https://github.com/Nargonath/cra-build-watch/blob/cd8ddd765f06c9d2c0587c41995b52dd9daa38ad/scripts/index.js#L13 to use the previous `webpack.config` file path or your new version. To see an example of how it is used: https://github.com/Nargonath/cra-build-watch/blob/cd8ddd765f06c9d2c0587c41995b52dd9daa38ad/scripts/index.js#L55

We also need to check that the array indexes have not changed and remain the same in the second line of code I linked.